src/uart.toit

```diff
@@ -0,0 +1,277 @@
+// Copyright (C) 2024 Toitware ApS. All rights reserved.
+// Use of this source code is governed by an MIT-style license that can be
+// found in the LICENSE file.
+
+import binary show LITTLE-ENDIAN
+import encoding.base64
+import encoding.json
+import encoding.ubjson
+import gpio
+import log
+import uart
+import reader show BufferedReader Reader
+import system
+
+import .jaguar
+
+class EndpointUart implements Endpoint:
+  config_/Map
+  logger/log.Logger
+
+  constructor --config/Map --logger/log.Logger:
+    config_ = config
+    this.logger = logger.with-name "uart"
+
+  run device/Device:
+    print "running uart endpoint"
+    logger.debug "running uart endpoint"
```

Comment:
Not sure you need the uart part in the message. The logger already has the tag. This also applies to the http one.

Maybe 'starting' is also better than 'running'.